cross/sudo/PLIST

```diff
@@ -0,0 +1,36 @@
+bin:bin/sudo
+bin:bin/sudoedit
+bin:bin/sudoreplay
+bin:include/sudo_plugin.h
+bin:libexec/sudo/group_file.la
+bin:libexec/sudo/group_file.so
+bin:libexec/sudo/libsudo_util.la
+bin:libexec/sudo/libsudo_util.so
+bin:libexec/sudo/libsudo_util.so.0
+bin:libexec/sudo/libsudo_util.so.0.0.0
+bin:libexec/sudo/sudoers.la
+bin:libexec/sudo/sudoers.so
+bin:libexec/sudo/sudo_noexec.la
+bin:libexec/sudo/sudo_noexec.so
+bin:libexec/sudo/system_group.la
+bin:libexec/sudo/system_group.so
+bin:sbin/visudo
+shr:share/doc/sudo/ChangeLog
```

Comment:
Remove `man`, `doc` and maybe `examples` too (if the examples are needed to get the package working, they might fit better in a `/var` location)